Xamarin.Forms.Core.UnitTests/MultiBindingTests.cs

```diff
@@ -71,8 +63,7 @@ public void TestChildOneWayOverridesMultiTwoWay()
 			Assert.AreEqual(oldName, label.Text);
 			Assert.AreEqual(oldName, group.Person1.FullName);
 
-			label.Text = $"{oldFirstName.ToUpper()} {oldMiddleName} {oldLastName.ToUpper()}";
```

Comment:
setting the label.Text directly is supposed to remove the MultiBinding. the fact that some changes were still happening after that indicates some references weren't cleared

Reply:
Ah good catch. The proxy is still hanging around listening for the change on the target property and updating the source even after the target binding is cleared.